File: 0058-length-of-last-word/0058-length-of-last-word.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int ans=0;
        bool f=false;
        for(int i=s.size()-1;i>=0;i--){
            if(s[i]!=' '){f=true;ans++;}
            if(s[i]==' '&&f)return ans;
        }
        return ans;
    }
};

File: 0058-length-of-last-word/0058-length-of-last-word.java
class Solution {
    public int lengthOfLastWord(String s) {
        int ans = 0;
        boolean f = false;
        for (int i = s.length() - 1; i >= 0; i--) {
            if (s.charAt(i) != ' ') {
                f = true;
                ans++;
            }
            if (s.charAt(i) == ' ' && f)
                return ans;
        }
        return ans;
    }
}